Resources/Views/Hub/home.leaf

```diff
@@ -0,0 +1,257 @@
+#extend("base"):
+    #export("title", "SwiftLeeds")
+        
+    #export("content"):
+
+    <header class="header navbar navbar-expand-lg navbar-dark bg-dark border-bottom border-light shadow-sm fixed-top">
+        <div class="container px-3">
+          <a href="/" class="navbar-brand">
+            <img src="/img/logo_horizontal_dark.svg" width="120" alt="SwiftLeeds">
+          </a>
+        </div>
+      </header>
+
+
+      <!-- Page content -->
+      <section class="container pt-5">
+        <div class="row">
+
+  
+          <!-- Sidebar (User info + Account menu) -->
+          <aside class="col-lg-3 col-md-4 border-end pb-5 mt-n5">
+            <div class="position-sticky top-0">
+              <div class="text-center pt-5">
+                <div class="d-table position-relative mx-auto mt-2 mt-lg-4 pt-5 mb-3">
+                  <img src="#(ticket.imageUrl)" class="d-block rounded-circle" width="120" alt="#(ticket.name)">
+                </div>
+                <h2 class="h5 mb-1">#(ticket.name)</h2>
+                
+                #if(false):
+                #if(ticket.purchaseTalkshowUrl):
+                <a href="#(ticket.purchaseTalkshowUrl)" target="_blank" class="badge bg-danger shadow-danger">Missing Talkshow</a>
+                #else:
+                <span class="badge bg-success shadow-success">Valid Talkshow Ticket</span>
+                #endif
+                #endif
+                
+                <p class="mt-3 fs-sm mb-3 pb-3">#(ticket.email)</p>
+                
+                <button type="button" class="btn btn-secondary w-100 d-md-none mt-n2 mb-3" data-bs-toggle="collapse" data-bs-target="#ticket-menu">
+                  <i class="bx bxs-user-detail fs-xl me-2"></i>
+                  Ticket Hub Menu
+                  <i class="bx bx-chevron-down fs-lg ms-1"></i>
+                </button>
+                
+                <div id="ticket-menu" class="list-group list-group-flush collapse d-md-block">
+                #if(false):
+                  <a href="#info" data-scroll data-scroll-offset="70" class="list-group-item list-group-item-action d-flex align-items-center active">
+                    <i class="bx bx-cog fs-xl opacity-60 me-2"></i>
+                    Personal Information
+                  </a>
+                  <a href="#merch" data-scroll data-scroll-offset="70" class="list-group-item list-group-item-action d-flex align-items-center">
+                    <i class="bx bxs-t-shirt fs-xl opacity-60 me-2"></i>
+                    Merch Preferences
+                  </a>
+                #endif
+                
+                #if(count(dropInSessions) > 0):
+                  <a href="#dropin" data-scroll data-scroll-offset="70" class="list-group-item list-group-item-action d-flex align-items-center">
+                    <i class="bx bx-chat fs-xl opacity-60 me-2"></i>
+                    Drop-in Sessions
```

Comment:
Can we change all these references "Drop-In" it's how we format it on the website and I think it just looks better 👍🏼

Reply:
It was easier to just write the copy for you:

> Drop-In sessions enable you to have a one-to-one conversation with one of our fantastic hosts below. Each session covers a unique subject, such as your apps, career, or anything else you want to discuss.

> Drop-In sessions are free, and one session is included in your ticket cost. However, slots are limited and offered on a first-come, first-served basis, so why not book a slot below?